.cspell.json

```diff
@@ -1111,6 +1111,7 @@
     "pycodestyle",
     "pydevproject",
     "pyflake",
```

Comment:
Should we remove this? :thinking:

Reply:
It exists from first commit.
https://github.com/tier4/autoware-spell-check-dict/commit/6164926893becaf1b37b15dd87ddb0ef4dfbfb72

It think it's spell miss and we can replace it.

Reply:
Added in https://github.com/tier4/autoware-spell-check-dict/commit/6164926893becaf1b37b15dd87ddb0ef4dfbfb72 (almost initial commit).